packages/web-components/src/components/va-promo-banner/va-promo-banner.css

```diff
@@ -0,0 +1,122 @@
+:host {
+  z-index: 1;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  background-color: var(--color-gray-lightest);
+  color: var(--color-link-default);
+  font-weight: 700;
+}
+
+.va-banner-body {
+  display: table;
+  width: 100%;
+  max-width: 100rem;
+  margin: 0 auto;
+}
+
+.va-banner-icon {
+  width: 5.4rem;
+  margin-right: 0.8rem;
+  padding: 0.8rem 0;
+  display: none;
+}
+
+.va-banner-icon-contents {
+  display: inline-block;
+  height: 4.2rem;
+  position: relative;
+  vertical-align: middle;
+  width: 4.2rem;
+  margin-left: 0.5rem;
+}
+
+.va-banner-icon-contents i {
+  position: absolute;
+  text-align: center;
+  width: 100%;
+  font-size: 2.15rem;
+  background-color: var(--color-white);
+  border-radius: 50%;
+  height: 100%;
+}
+
+.va-banner-icon-contents i::before {
+  position: relative;
+  top: 0.8rem;
+}
+
+.va-banner-icon-contents i.announcement::before {
+  content: '\F0A1'; /* fa-bullhorn */
+}
+
+.va-banner-icon-contents i.news::before {
+  content: '\F1EA'; /* fa-newspaper */
+}
+
+.va-banner-icon-contents i.email-signup::before {
+  content: '\F0E0'; /* fa-envelope */
+}
+
+.va-banner-content {
+  padding: 0.8rem;
+  line-height: 2rem;
+}
+
+.va-banner-content-link {
+  display: inline-block;
+  text-decoration: none;
+  color: var(--color-link-default);
+}
```

Comment:
The React implementation has `text-decoration: underline` on `:hover`. We probably want to add that.

Reply:
Added good catch